app/models/channel.rb

```diff
@@ -281,6 +284,20 @@ def register_channel_for_promo
     RegisterChannelForPromoJob.new.perform(channel: self)
   end
 
+  def set_channel_banner
+    headline = I18n.t 'banner.headline'
+    tagline = I18n.t 'banner.tagline'
+    channel_banner = SiteBanner.create(
+      publisher_id: self.publisher_id,
+        channel_id: self.id,
+        title: headline,
```

Comment:
Fewer lines the better IMO. e.g. `title: I18n.t 'banner.headline'` is the preferred style.